tests/dualtor/test_server_failure.py

```diff
@@ -24,7 +24,7 @@ def test_server_down(duthosts, tbinfo, rand_selected_interface, simulator_flap_c
         
     upper_tor = duthosts[tbinfo['duts'][0]]
     lower_tor = duthosts[tbinfo['duts'][1]]
-    PAUSE_TIME = 5
+    PAUSE_TIME = 10
```

Comment:
Instead of just increase sleep time, please change line 32 and line 36-40 to wait_until().

Reply:
Updated accordingly.